core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -134,6 +134,12 @@ object StorageTool extends Logging {
       case None => Option(config.originals.get(ReplicationConfigs.INTER_BROKER_PROTOCOL_VERSION_CONFIG)).
         foreach(v => formatter.setReleaseVersion(MetadataVersion.fromVersionString(v.toString)))
     }
+    Option(namespace.getList[String]("feature")).foreach(
```

Comment:
Previously we didn't allow for setting `release_version` and `feature` flags. This was more of a problem when release_version set all the corresponding features. Here, we may see metadata version in both places though -- which takes precedence?

Reply:
Features in general do take their defaults from `--release-version`. KIP-849 is just a bit special because it requires a special configuration and we didn't want to make a breaking change. Then `--feature` takes priority over `--release-version`.

Reply:
Makes sense thanks!

Reply:
Maybe when we make the other features production ready, we can add some tests for that too. 👍